python_apps/media-monitor/install/airtime-media-monitor.conf

```diff
@@ -0,0 +1,15 @@
+description "Airtime Media Monitor"
+author "help@sourcefabric.org"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+
+setuid WEB_USER
```

Comment:
If all of these files are being templated with WEB_USER, should we rename them to blah.conf.template instead, to make it clearer?